packages/publisher/bin/publish

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env node
+
+const sh = require('shelljs')
+const { docopt } = require('docopt')
+const { gt } = require('semver')
+const { resolve } = require('path')
+
+const DOC = `
+Publish packages only if their version is newer than npm's.
+
+Usage:
+    publish [options]
+    publish -h | --help
+
+Options:
+    --npm-token=STRING   Update ~/.npmrc with npm token for authentication
+    -h --help            Show this screen
+`
+
+;(() => {
```

Comment:
I prefer it over the following and as the script evolves, it's easier to break down into helper functions because variables aren't all on the global scope.
```
function main() { ... }
main()
```